<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gestion des employés</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>991</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>gerer la liste des employés</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>201</width>
        <height>521</height>
       </rect>
      </property>
      <property name="title">
       <string>ajouter employé</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ajout_nom">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Prenom:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ajout_prenom">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>date de naissanse:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>330</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Role:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="ajout_role">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>350</y>
         <width>171</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Responsable resourse humaine</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>chauffeur</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>380</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Login:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ajout_login">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>400</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>430</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>password:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ajout_password">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>450</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="ajout_date">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>171</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_ajout">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>480</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>ajouter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>CIN:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ajout_cin">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>8</number>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Numero telephone:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ajout_num">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>8</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>280</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Adresse email:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ajout_email">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>300</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>0</y>
        <width>551</width>
        <height>521</height>
       </rect>
      </property>
      <property name="title">
       <string>affichier/rechercher employé</string>
      </property>
      <widget class="QGroupBox" name="groupBox_9">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>20</y>
         <width>251</width>
         <height>131</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>CIN d'employé:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_14">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>221</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>221</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Affichier l'emploi du temp pour se semain</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>251</width>
         <height>131</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QPushButton" name="pushButton_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>rechercher</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_aff">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>80</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>affichier tout</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_15">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>221</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>221</width>
          <height>22</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>rechercher par CIN</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>rechercher par nom</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>rechercher par prenom</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>rechercher par role</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>430</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>trié le tableau:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>450</y>
         <width>191</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>par ordre alphebetique de prenom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>par ordre alphebetique de nom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>par ordre alphebetique de login</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>par ordre alphebetique de email</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButton_11">
       <property name="geometry">
        <rect>
         <x>404</x>
         <y>450</y>
         <width>121</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Trier le tableau</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>430</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>ordre:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_8">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>450</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>croissant</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>décroissant</string>
        </property>
       </item>
      </widget>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>511</width>
         <height>261</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>0</y>
        <width>191</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>modifier un employé</string>
      </property>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>CIN d'employé a modifier:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="cin_mod">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>attribut a modifier:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="attribut_mod">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>171</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Nom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Prenom</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CIN</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Role</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Email</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Telephone</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>sa nouveau valeur:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="mod_val">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_mod">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>170</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_supp">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>210</y>
        <width>191</width>
        <height>311</height>
       </rect>
      </property>
      <property name="title">
       <string>msg d'erreur:</string>
      </property>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>171</width>
         <height>261</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>460</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>exporter</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>811</width>
        <height>411</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>987</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
